specification/monitor/resource-manager/Microsoft.Insights/stable/2023-10-01/metrics_API.json

```diff
@@ -0,0 +1,610 @@
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Insights/metrics": {
+      "get": {
+        "tags": [
+          "Metrics"
+        ],
+        "operationId": "Metrics_ListAtSubscriptionScope",
+        "description": "**Lists the metric data for a subscription**.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./commonMonitoringTypes.json#/parameters/RegionParameter"
+          },
+          {
+            "$ref": "#/parameters/TimespanParameter"
+          },
+          {
+            "$ref": "./commonMonitoringTypes.json#/parameters/IntervalParameter"
+          },
+          {
+            "$ref": "./commonMonitoringTypes.json#/parameters/MetricNamesParameter"
+          },
+          {
+            "$ref": "./commonMonitoringTypes.json#/parameters/AggregationsParameter"
+          },
+          {
+            "$ref": "./commonMonitoringTypes.json#/parameters/TopParameter"
+          },
+          {
+            "$ref": "./commonMonitoringTypes.json#/parameters/OrderByParameter"
+          },
+          {
+            "$ref": "#/parameters/FilterParameter"
+          },
+          {
+            "$ref": "./commonMonitoringTypes.json#/parameters/RollUpByParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionScopeResultTypeParameter"
+          },
+          {
+            "$ref": "./commonMonitoringTypes.json#/parameters/MetricNamespaceParameter"
+          },
+          {
+            "$ref": "./commonMonitoringTypes.json#/parameters/AutoAdjustTimegrain"
+          },
+          {
+            "$ref": "#/parameters/ValidateDimensions"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./commonMonitoringTypes.json#/definitions/ErrorContract"
+            }
+          },
+          "200": {
+            "description": "Successful request to get the list of metric values.",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionScopeMetricResponse"
+            }
+          }
+        },
+        "x-ms-odata": "#/definitions/MetadataValue",
+        "x-ms-examples": {
+          "Get subscription level metric data": {
+            "$ref": "./examples/GetMultiResourceMetric.json"
+          },
+          "Get subscription level metric metadata": {
+            "$ref": "./examples/GetMultiResourceMetricMetadata.json"
+          }
+        }
+      },
+      "post": {
+        "tags": [
```

Comment:
post cannot be defined on a resource collection. 

valid api paths for POST actions are '

/subscriptions/{subscriptionId}/providers/Microsoft.Insights/metrics/{metricName}/{actionName}

OR
/subscriptions/{subscriptionId}/providers/Microsoft.Insights/{actionName}